gitpwnd/bootstrap.py.template

```diff
@@ -177,9 +177,17 @@ def find_root_of_git_repo(path):
         else:
             cur_dir = os.path.join(cur_dir, "..")
 
-# TODO: Iterate through all dirs in base_dir and find all the git repos
+def is_git_directory(path):
+    exit_code = subprocess.call(["git", "status"], cwd=path, stdout=open(os.devnull, 'w'), stderr=subprocess.STDOUT)
+    return (exit_code == 0)
+
 def find_git_repos(base_dir):
-    return []
+    git_repos = []
+    for directory in next(os.walk())[1]:
```

Comment:
It doesn't look like you're using the function's argument,`base_dir`, when iterating over files, am I reading this wrong?